app/views/posts/show.html.erb

```diff
@@ -6,7 +6,7 @@
   </p>
   <p><%= @post.text %></p>
   <div class="align-self-end">
-    <%= form_with url: post_likes_path(@post), method: :post do |f| %>
+    <%= form_with url: user_post_likes_path(current_user, @post), method: :post do |f| %>
```

Comment:
- [ ] Another error is happening when I wants to like a post,  the post is nil
<img width="1081" alt="image" src="https://github.com/zeff96/blog_app/assets/65068771/6b36355f-bc9d-400f-95a5-8481807b9436">